README.md

```diff
@@ -5,6 +5,7 @@
     - [Upgrading Plugins to work with OpenSearch](#upgrading-plugins-to-work-with-opensearch)
     - [Installing Plugins](#installing-plugins)
     - [Plugin Release Notes](#plugin-release-notes)
+    - [Licenses and License Headers ](#faq)
```

Comment:
Link is broken, it's not `faq`. I use one of the very many VSCode plugins for these, search for "automatic TOC"

Reply:
ugh, I use intellij on local. I think I should add this.
Nice catch, thanks.